pom.xml

```diff
@@ -202,6 +203,13 @@
         <version>${awaitility.version}</version>
       </dependency>
 
+      <dependency>
+        <!-- Eclipse Java compiler -->
+        <groupId>org.eclipse.jdt</groupId>
+        <artifactId>ecj</artifactId>
```

Comment:
- [ ] Does this need to be marked as "optional" to avoid users classloading this on Java 11 by mistake, or does it not actually matter here?